### PR TITLE
[android] material dialogs update

### PR DESCRIPTION
**Pull request by marcelinkaaa**
Since we changed the min API and are now closer to supporting only material I wanted to update the dialogs, especially its buttons so its consistent across the app - as before for ex. we would use fully material dialogs in preferences (and for social dialog) but custom dialogs (with different buttons elsewhere). 

The issue with custom preference dialog I had before is fixed. 

dialogs

### CODE REVIEW

**Comment by gubatron** [comment]
I can help with this, but I want to continue the work on transfers first, more important.

**Comment by gubatron** [comment]
do you still need help with this or can it be merged?

**Comment by marcelinkaaa** [comment]
No help needed unless you see something you would like me to adjust. Works well.

File: android/src/com/frostwire/android/gui/views/AbstractPreferenceFragment.java [hunk 1]
```diff
@@ -106,7 +106,8 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
             onClick(null, DialogInterface.BUTTON_NEGATIVE);
 
             AlertDialog.Builder builder = new AlertDialog.Builder(context)
-                    .setTitle(get("mDialogTitle"))
+                    // Title set directly in a custom dialog
+                    //.setTitle(get("mDialogTitle"))
```
Comment (gubatron) [inline android/src/com/frostwire/android/gui/views/AbstractPreferenceFragment.java]: if these won't be here, let's delete these comments

File: android/src/com/frostwire/android/gui/views/preference/CustomSeekBarPreference.java [hunk 2]
```diff
@@ -222,6 +225,10 @@ public void onStopTrackingTouch(SeekBar seekBar) {
             mCurrentValueTextView = view.findViewById(R.id.dialog_preference_seekbar_with_checkbox_current_value_textview);
             mUnlimitedCheckbox = view.findViewById(R.id.dialog_preference_seekbar_with_checkbox_unlimited_checkbox);
             Bundle arguments = getArguments();
+
+            mDialogTitleTextView = view.findViewById(R.id.dialog_preference_seekbar_with_checkbox_title);
```
Comment (gubatron) [inline android/src/com/frostwire/android/gui/views/preference/CustomSeekBarPreference.java]: 👍

**Pull request merged**